mobile/src/main/java/org/openhab/habdroid/util/Util.java

```diff
@@ -146,4 +146,14 @@ public static void setActivityTheme(@NonNull final Activity activity, String the
         }
         activity.setTheme(themeRes);
     }
+
```

Comment:
Why not pass cause as `Class<?> causeType` and compare `error.getCause()` to that? Saves the instance creations...

Reply:
Yes, thats exactly what I'm currently trying. How can I call the method then?

Reply:
Nevermind. got it